app/controllers/books_controller.rb

```diff
@@ -69,6 +69,7 @@ def clear_list
 
   def show_list
     @books = Book.where(selected: true)
+    @multi_loan_available = is_librarian? && (@books - Book.available_to_loan).empty? && (@books.length < 6)
```

Comment:
Ensures all books in list are available to loan, and that there are less than 6. We should reference the MAX_LOANS constant from user.rb here. Is that accessible from here?